File: Backend/ApiUniversidad/ApiUniversidad/Models/Alumno.cs
using System;
using System.Collections.Generic;

namespace ApiUniversidad.Models;

public partial class Alumno
{
    public Guid Id { get; set; } = Guid.NewGuid();

    public string Nombre { get; set; } = null!;

    public string Apellido { get; set; } = null!;

    public string Legajo { get; set; } = null!;

    public Guid IdRol { get; set; }

    public DateTime FechaAlta { get; set; }

    public virtual ICollection<AlumnosPorCurso> AlumnosPorCursos { get; set; } = new List<AlumnosPorCurso>();

    public virtual Role IdRolNavigation { get; set; } = null!;
}

File: Frontend/ApiUniversidad/UniversidadAngular/src/app/models/alumno.ts
import { AlumnosPorCurso } from "./alumnos-por-curso";
import { Rol } from "./rol";

export class Alumno {

    id?: string;
    nombre: string = '';
    apellido: string = '';
    legajo: string = '';
    idRol: string = '';
    fechaAlta: Date = new Date();
    alumnosPorCurso: AlumnosPorCurso[] = [];
    idRolNavigation: Rol = new Rol();
    
}